src/Conversations/Conversation.php

```diff
@@ -22,11 +22,11 @@
 
 class Conversation implements Extractable, Hydratable
 {
-    use HydratesData,
-        ExtractsData,
-        HasPartiesToBeNotified,
-        IncludesThreadDetails,
-        HasCustomer;
+    use HydratesData;
+    use ExtractsData;
+    use HasPartiesToBeNotified;
+    use IncludesThreadDetails;
+    use HasCustomer;
```

Comment:
Got a bunch of complaints by CI on this kind of thing... not sure why it's suddenly caring about these and wasn't before...

Reply:
I think it might be the Symfony PHP CS Fixer ruleset may have changed? https://github.com/helpscout/helpscout-api-php/blob/master/.php_cs. Perhaps we've updated the dependencies recently? If we want to allow it we can adjust the ruleset in that config file like we did for some others, yoda style for example.

Reply:
On that note, I blew away my vendor directory and was suddenly seeing this issue locally.  I've added `'single_trait_insert_per_statement' => false,` and reverted all the `use` usages I added so we're back to multiple traits per `use` statement.